### PR TITLE
dockerfile: add small comment explaining METADATA=true

### PR DESCRIPTION
**Pull request by roobre**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,8 @@ RUN mv nri-kube-events-${TARGETOS}-${TARGETARCH} nri-kube-events
 
 USER nri-kube-events
 
-# Enable custom attributes decoration in the infra SDK
+# Enable decorating events from NRI_ env vars, such as `foo: bar` from NRI_FOO=bar.
+# https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/toolset/args.md#bundled-arguments
```
Comment (paologallinaharbur) [inline Dockerfile:24]: I would add a link as well to https://docs.newrelic.com/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration/#custom-attributes

to understand how it is used

**Review by paologallinaharbur — APPROVED** [review]

**Comment by roobre** [comment]
We still need to check whether `NRI_attribute` is enough, or `NRI_KUBE_EVENTS_attribute` is required as the docs seem to hint.

**Comment by roobre** [comment]
We still need to check whether `NRI_attribute` is enough, or `NRI_KUBE_EVENTS_attribute` is required as the docs seem to hint.

**Comment by roobre** [comment]
Closing as this has become stale, plus this mechanism is not going to be used in the future.

**Pull request closed without merging**